src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner.tsx

```diff
@@ -95,6 +94,13 @@ export const DashboardTopBanner = (props: DashboardTopBannerProps) => {
           )}
         </Button>
       ),
+      learnMore: (
+        <a href="https://mozilla.org/products/monitor/how-it-works/">
```

Comment:
Note: this 404s, but i assume is something that will be added by SuMo/Bedrock team?

Reply:
This is provided by bedrock. Yes I believe it's being worked on.